### PR TITLE
Clarify #backgroundColor re: block, inline and region semantics

### LINKED ISSUE #255: Clarify #backgroundColor regarding inclusion of block, inline, region semantics.
See also https://github.com/w3c/ttml2/issues/455.

### PR DESCRIPTION
**Pull request by palemieux**
Close #255

### CODE REVIEW

**Review by nigelmegitt — APPROVED** [review]
Looks good to me. Probably doesn't need rebuilt `schema.zip` files, but they'll get rebuilt anyway later.

File: spec/ttml1.xml [hunk 1]
```diff
@@ -8970,6 +8970,10 @@ attribute
 and (2) is capable of displaying or generating an output display
 signal that distinguishes between at least sixteen (16) values of color,
 including all primary and secondary colors of the SRGB color space.</p>
+<note role="explanation">
+<p>Support for this feature is intended to imply support for the following features: <code>#backgroundColor-block</code>,
```
Comment (skynavga) [inline spec/ttml1.xml:8974]: Change "is intended to imply" to "implies".

**Review by skynavga — CHANGES_REQUESTED** [review]

File: spec/ttml1.xml (hunk 1 above)
Comment (palemieux) [reply spec/ttml1.xml:8974]: @skynavga The form "intended to imply support" is used throughout feature definitions. Why do it differently here?

Comment (skynavga) [reply spec/ttml1.xml:8974]: In that case, I withdraw my comment.

Comment (palemieux) [reply spec/ttml1.xml:8974]: @skynavga Please approve the PR then.

**Review by skynavga — APPROVED** [review]

**Comment by nigelmegitt** [comment]
Built version available at https://rawgit.com/w3c/ttml1/issue0255-backgroundColor-build/index.html

**Pull request merged**